MykSuite/src/main/java/com/infomaniak/lib/myksuite/ui/views/MyKSuiteUpgradeBottomSheetDialog.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Infomaniak Mail - Android
+ * Copyright (C) 2025 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.lib.myksuite.ui.views
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import com.infomaniak.lib.myksuite.ui.screens.MyKSuiteUpgradeBottomSheet
+import com.infomaniak.lib.myksuite.ui.screens.components.ButtonType
+
+@OptIn(ExperimentalMaterial3Api::class)
+class MyKSuiteUpgradeBottomSheetDialog : BottomSheetDialogFragment() {
+
+    private val navigationArgs: MyKSuiteUpgradeBottomSheetDialogArgs by navArgs()
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
+        return ComposeView(requireContext()).apply {
+            setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+            setContent {
+                MyKSuiteUpgradeBottomSheet(
+                    style = ButtonType.Mail,
+                    onDismissRequest = this@MyKSuiteUpgradeBottomSheetDialog.findNavController()::popBackStack,
+                    customFeatures = { navigationArgs.upgradeFeatures.toList() }
```

Comment:
`customFeatures`'s callback doesn't need to be `Composable`. As a matter of fact, I'm not even sure you need it to be a callback at all, this value is static and never changes right? Or is this some mutableState in disguise?

Reply:
I already removed the callback in precedent PR, but anyway this whole parameter will be removed in the futur so it's not important